docs/weak.rst

```diff
@@ -418,7 +418,7 @@ on top of basic runtime support.
 The first set of use cases will require more direct language support.
 To that end, I propose adding two new variable attributes,
 :code:`@weak` and :code:`@unowned`.  I also propose a small slate of
-new features which directly address the problem of capturing a value
```

Comment:
This doesn't need a comma
http://www.grammarly.com/answers/questions/17353-use-of-comma-before-which-and-but/